docs/website/assets/sass/custom.sass

```diff
@@ -60,12 +60,39 @@
 .article
   padding-bottom: 5rem
 
+  .hero
+    width: 75%
+
+    +touch
+      width: 100%
+    
+    .hero-body
+      padding: 2rem 1.5rem 0 3rem
+
+      +touch
+        padding: 0 2rem
+
+  .section
+    padding: 0rem 3rem
+
+    +touch
+      padding: 0rem 2rem
+
   .content
     width: 75%
 
     +touch
       width: 100%
 
+  .icon-edit
+    float: right
+    padding-top: 2rem
+    line-height: 0.5rem
+    text-align: center
+
+    span
+      font-size: 0.7rem !important
```

Comment:
🤔 what's the need for `!important`?

Reply:
You are right, it is not needed. Amended the changes to pull request